src/components/ProfileForm/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { reduxForm, Field } from 'redux-form/immutable';
+import PropTypes from 'prop-types';
+
+import CustomInput from '../common/CustomInput';
+import CustomButton from '../common/CustomButton';
+import { validations, editProfile } from '../../utils/constraints';
+import { black } from '../../constants/styleConstants';
+import styles from './styles';
+
+const ProfileForm = ({
+  handleSubmit, error, submitting, editProfile
+}) => (
+  <View>
```

Comment:
Have you checked if a keyboardAwareView is necessary here?

Reply:
I haven't still but because it's not part of the logout functionality.